build/DependencyHelper.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.DependencyModel;
+using Newtonsoft.Json.Linq;
+
+namespace Build
+{
+    public static class DependencyHelper
+    {
```

Comment:
Reference class in Host - [Link](https://github.com/Azure/azure-functions-host/blob/b612457d32902cd4b6b9c5b32b5eecb20240549c/src/WebJobs.Script/Description/DotNet/DependencyHelper.cs#L15)